packages/interface/src/components/interface-skeleton/style.scss

```diff
@@ -82,9 +82,13 @@ html.interface-interface-skeleton__html-container {
 	// On Mobile the header is fixed to keep HTML as scrollable.
 	// Beyond the medium breakpoint, we allow the sidebar.
 	// The sidebar should scroll independently, so enable scroll here also.
-
 	overflow: auto;
 
+	// On Safari iOS on smaller viewports lack of a z-index causes the background
+	// to "bleed" through the header.
+	// See https://github.com/WordPress/gutenberg/issues/32631
+	z-index: z-index(".interface-interface-skeleton__body");
```

Comment:
Oh, gotcha the diff was hiding that this is being applied to `.interface-interface-skeleton__content`.

Reply:
Yeh I'm sorry. It was unclear because I forgot about the naming of this CSS var `.interface-interface-skeleton__body`. It should be `.interface-interface-skeleton__content`. Looks like @noisysocks sorted it after he merged early.